test/src/nyc-test.js

```diff
@@ -430,4 +434,25 @@ describe('nyc', function () {
       return done()
     })
   })
+
+  describe('cache', function () {
+    it('handles collisions', function (done) {
+      var nyc = new NYC({cwd: fixtures})
+      nyc.clearCache()
+
+      var args = isWindows ? [bin] : [bin, process.execPath]
```

Comment:
This makes me thinks we are making a mistake in `mungeArgs` when it coms to windows.